meta-ostro/recipes-swupd/swupd-client/swupd-client/efi-combo-trigger.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Updates the EFI combo application
+BindsTo=update-triggers.target
+
+[Install]
+WantedBy=update-triggers.target
+
+[Service]
+Type=oneshot
+ExecStart=/usr/bin/efi_combo_updater
```

Comment:
ok, as it is it's better to move to shell. we can review that (I filled an issue as per above) later.

on clear/arjan, I'd say he agreed with you, so send an ACK with actual changes, maybe a patch/PoC. He says basically "this is the case today", so if we miss a small detail, let them know.